File: concept/10_array/problem/unique_number/unique_number.dart
void main() {
  /* Unique number
    Array of Lenth N.
    N = 2M+1 
  */

  List<int> data = [4, 4, 1, 6, 6];
  int ans = 0;
  for (var i = 0; i < data.length; i++) {
    ans = ans ^ data[i];
  }
  print(ans);
}

File: concept/10_array/problem/unique_number/unique_number.cpp
#include <iostream>
using namespace std;

void unique(int arr[],int size){
    int ans = 0;
    for (int i = 0; i < size; i++){
        ans = ans^arr[i];
        /* code */
    }
        cout<<ans<<" ";
    return;
}

int main(){


    /*
    Array of Lenth N.
    N = 2M+1 
    */


    cout<< "Unique number"<< endl;
   int arr[7] = {4,5,3,4,3};

   unique(arr,7);

   return 0 ;
    
}
